Chatify/chat/api.py

```diff
@@ -70,7 +81,7 @@ def post(self, request, *args, **kwargs):
 
 class OnlineUsersAPI(APIView):
     def get(self, request):
-        data = User.objects.filter(is_active=True).order_by("-id")
+        data = User.objects.filter(is_online=True).order_by("-id")
         return JsonResponse({"UserData": list(UserSerializer(data, many=True).data)})
 
```

Comment:
here, if we only want to show online user then in user model field name is is_online that's why i put is_online true.